media.graphql

```diff
@@ -0,0 +1,185 @@
+# This GraphQL schema defines the META Media types and queries.
+#
+# It consists of the following main entities:
+#
+# MusicPerformer
+# MusicComposer
+# RecordLabel
+# MusicPublisher
+# MusicRecording
+# MusicWork
+# MusicRelease
+# MusicProduct
+#
+# Each entity has a set of widely used identifiers which are used to uniquely
+# identify them:
+#
+# ISNI - International Standard Name Identifier
+# IPI  - Interested Parties Information
+# DPID - DDEX Party Identifier
+# ISRC - International Standard Recording Code
+# ISWC - International Standard Musical Work Code
+# GRid - Global Release Identifier
+# UPC  - Universal Product Code
+#
+# Other entity properties (like performer name or recording title) are linked
+# to these identifiers by various sources in the META network.
+
+#
+# --- Main Entities ---
+#
+type MusicPerformer {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  products:   [MusicProductLink]
+  releases:   [MusicReleaseLink]
+  recordings: [MusicRecordingLink]
+  labels:     [RecordLabelLink]
+}
+
+type MusicComposer {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  works:      [MusicWorkLink]
+  publishers: [MusicPublisherLink]
+}
+
+type RecordLabel {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  products:   [MusicProductLink]
+  releases:   [MusicReleaseLink]
+  recordings: [MusicRecordingLink]
+  performers: [MusicPerformerLink]
+}
+
+type MusicPublisher {
+  identifiers: [PartyIdentifier]
+
+  name: StringValue
+
+  composers: [MusicComposerLink]
+  works:     [MusicWorkLink]
+}
+
+type MusicRecording {
+  isrc: ID
+
+  title: StringValue
+
+  products:   [MusicProductLink]
+  releases:   [MusicReleaseLink]
+  performers: [MusicPerformerLink]
+  labels:     [RecordLabelLink]
+  works:      [MusicWorkLink]
+}
+
+type MusicWork {
+  iswc: ID
```

Comment:
"The ISWC identifies musical works by reference to the title of a work and its creators (including composers, authors, and arrangers).   It **does not** however serve to identify instances of use of the work in manifestations, such as publications, recordings or broadcasts). Other codes exist for such purposes: the International Standard Recording Code (ISRC) for sound recordings; the International Standard Code for Text (ISTC) or any other type of performance associated with the musical work."
This is taken cisac http://www.cisac.org/What-We-Do/Information-Services/International-Identifiers. 
So I am not sure about the musical work which identify by iswc can link to recording.

Reply:
I'm not sure what you mean, we are using ISWC precisely to identify works, not recordings of the work?

Reply:
ok. how you  link to the "recording" in the "work" to a specific work  ?
Does ERN provid this link by iswc ?

Reply:
We are currently using MusicBrainz: https://github.com/meta-network/go-meta/pull/37